### DISCUSSION TITLE
Resize document pages that have different sizes and orientations

### DISCUSSION THREAD

**Discussion by patdunlavey** [tavinus/pdfScale#43] [Q&A]
I have pdf files originally created from scans of photographic slides. Some of the pages are in portrait orientation, some in landscape orientation. The page sizes need to be increased from their original while keeping their original orientations `*`. I can use the `--resize <paper>` option, however it automatically rotates pages to best-fit the `<paper>`, and leaves them rotated in the output.

I also have cases where I need to resize oversized pages down, while leaving other pages unchanged.

Is there a way to resize pages of a pdf using pdfScale, maybe to a percentage of their original size, without having them automatically rotate? Is there a way to apply page resizing to individual pages, rather than the same value to all pages at once?

If not, can you recommend another tool? 

`*` Why? Because these pdf pages are utilized by [cantaloupe](https://cantaloupe-project.github.io/), using the [Apache PdfBoxProcessor](https://cantaloupe-project.github.io/manual/3.2/processors.html#PdfBoxProcessor) to convert them to image files to serve over the web. PdfBoxProcessor bases its rasterization on the page size in the pdf. Since these slides have very small page sizes (in the range of 1 inch), even high resolution slide scans (scanned at 3,000 dpi and up) end up being rendered with pixel dimensions on the order of 150 x 100.

**Comment by tavinus** [comment]
Did you try messing with auto-rotation?
```
-a, --auto-rotate <mode>
             Setting for GS -dAutoRotatePages, defaults to 'PageByPage'
             Uses text-orientation detection to set Portrait/Landscape
             Modes: p, pagebypage  Auto-rotates pages individually
                    n, none        Retains orientation of each page
                    a, all         Rotates all pages (or none) depending
                                   on a kind of "majority decision"
```
Try NONE
`-a n`
